samples/k8s-existing-cluster-only-sample/README.md

```diff
@@ -0,0 +1,33 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Existing cluster only sample
+
+This folder contains samples that shows how ISV can use only existing cluster option in their create ui definition.
+
+## The resulting UI Definition will show up as follows:
+
+![Alt text](images/UI_Sample.PNG)
```

Comment:
Can you also add an example of Cluster Details image? Above image has only basic settings.